usaspending_api/common/cache_decorator.py

```diff
@@ -26,12 +26,13 @@ def process_cache_response(self, view_instance, view_method, request, args, kwar
             if not response.status_code >= 400 or self.cache_errors:
                 if self.cache_errors:
                     logger.error(self.cache_errors)
-                try:
-                    self.cache.set(key, response, self.timeout)
-                    response["Cache-Trace"] = "set-cache"
-                except Exception:
-                    msg = "Problem while writing to cache: path:'{p}' data:'{d}'"
-                    logger.exception(msg.format(p=str(request.path), d=str(request.data)))
+                if request.headers.get("X-Experimental-Api") != "elasticsearch":
```

Comment:
I don't think placing the conditional here satisfies AC #2 since an endpoint with ES functionality can still return a cached response.

Reply:
right you are, changes made